src/v2/Apps/Conversation/Components/OpenInquiryModalCTA.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from "react"
+import {
+  Button,
+  CheckCircleIcon,
+  Flex,
+  Text,
+  Link,
+  Separator,
+} from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useTracking } from "react-tracking"
+import { tappedMakeOffer } from "@artsy/cohesion"
+import styled from "styled-components"
+import { OpenInquiryModalCTA_conversation } from "v2/__generated__/OpenInquiryModalCTA_conversation.graphql"
+
+export const ShadowSeparator = styled(Separator)`
+  box-shadow: 0 -1px 1px rgba(50, 50, 50, 0.1);
+  width: 100%;
+  height: 0;
+`
+
+export interface OpenInquiryModalCTAProps {
+  openInquiryModal: () => void
+  conversation: OpenInquiryModalCTA_conversation
+}
+
+export const OpenInquiryModalCTA: React.FC<OpenInquiryModalCTAProps> = ({
+  openInquiryModal,
+  conversation,
+}) => {
+  const tracking = useTracking()
+
+  const handleOpenModal = () => {
+    tracking.trackEvent(tappedMakeOffer(conversation?.internalID ?? ""))
+    openInquiryModal()
+  }
+
+  return (
+    <>
+      <ShadowSeparator />
+      <Flex flexDirection="column" p={1}>
+        <Flex flexDirection="row">
+          <CheckCircleIcon mr={1} />
+          <Flex flexShrink={1}>
+            <Text color="black60" variant="xs" mb={1}>
+              Only purchases completed with our secure checkout are protected by{" "}
+              <Link target="_blank" href="/buyer-guarantee">
+                The Artsy Guarantee
+              </Link>
+              .
+            </Text>
+          </Flex>
+        </Flex>
+        <Button
+          size="medium"
+          variant="primaryBlack"
+          onClick={() => handleOpenModal()}
+        >
+          Make Offer
+        </Button>
+      </Flex>
+    </>
+  )
+}
+
+export const OpenInquiryModalCTAFragmentContainer = createFragmentContainer(
```

Comment:
the only change from OpenInquiryModalButton is we have only a Fragment and we control if we're going to render it or not from the caller, we don't need to check `isOfferableFromInquiry` here.